vassal-doc/src/main/readme-referencemanual/ReferenceManual/DoActionButton.adoc

```diff
@@ -48,22 +51,34 @@ If the expression begins the loop already false, the actions will not be perform
 The set of actions will be performed at least once.
 
-**Loop how many times? **An <<Expression.adoc#top,Expression>> that defines how many times to generate the set of actions (for a fixed loop).
+**Loop how many times **An <<Expression.adoc#top,Expression>> that defines how many times to generate the set of actions (for a fixed loop).
 
 *Loop while/until condition is true:* Enter the <<PropertyMatchExpression.adoc#top,Property Expression>> that controls how long to keep looping (for a while or until loop).
 
-*Perform this KeyStroke once before looping starts:* For all loop types, enter a HotKey or <<NamedKeyCommand.adoc#top,Named Key Command>> that will be performed once before the looping process commences.
+*Perform this Hotkey once before looping starts:* For all loop types, enter a <<NamedKeyCommand.adoc#top,Keystroke or Named Command>> that will be performed once before the looping process commences.
 
-*Perform this KeyStroke once after looping complete:* For all loop types, enter a HotKey or <<NamedKeyCommand.adoc#top,Named Key Command>> that will be performed once after the looping process finishes.
+*Perform this Hotkey once after looping complete:* For all loop types, enter a <<NamedKeyCommand.adoc#top,Keystroke or Named Command>> that will be performed once after the looping process finishes.
 
-**Create a Loop Index Property? **For all loop types, allows you to create a Property whose value will changes as the looping process proceeds.
+**Create a loop index property? **For all loop types, allows you to create a Property whose value will change as the looping process proceeds.
 
-*Index Property Name:* The name of the property that will be created.
+*Index property name:* The name of the property that will be created.
 
-*Index Property start value:* An <<Expression.adoc#top,Expression>> that sets the value of the index property prior to looping commencing.
+*Index property start value:* An <<Expression.adoc#top,Expression>> that sets the value of the index property prior to looping commencing.
 The expression must evaluate to a whole number.
 
-*Index Property increment value:* An <<Expression.adoc#top,Expression>> that defines how much will be added to the Index Property at the start of each loop.
+*Index property increment value:* An <<Expression.adoc#top,Expression>> that defines how much will be added to the Index Property at the start of each loop.
 The expression must evaluate to a whole number.
 
-image:images/DoActionButton.png[] +
+| image:images/DoActionButton.png[] +
+
+A complex example that does the following:
+
+. Send the Hotkey _StartHitting_
+. Set up a property name _Counter_ with a values of 10 and repeat the following steps, incrementing _Counter_ by 2 each time until _Counter_ exceeds the value of the property _MaxCount_
+.. Display the message 'Opennent Hit!!!' in the Chat area.
```

Comment:
Typos--"Opennent" should be "Opponent".  Also, this is the only entry in the list to have a period/full-stop at the end.